packages/bsky/src/index.ts

```diff
@@ -63,6 +64,10 @@ export class BskyAppView {
     const imgUriBuilder = new ImageUriBuilder(
       config.cdnUrl || `${config.publicUrl}/img`,
     )
+    const videoUriBuilder = new VideoUriBuilder({
+      playlistUrlPattern: `${config.publicUrl}/vid/%s/%s/playlist.m3u8`,
```

Comment:
should this be the config value instead (ie `config.videoPlaylistUrlPattern`) and only if that isn't set then it uses the `publicUrl` based pattern? a la the imgUriBuilder

Reply:
Ay, good catch, thanks!